src/datachain/lib/arrow.py

```diff
@@ -7,7 +7,9 @@
 from pyarrow.dataset import dataset
```

Comment:
Changes in this file are about saving struct fields as nested fields rather than as a single json field, which was failing when the struct included bytes objects.